providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/process/storage/connector/sync/SyncResponseModel.java

```diff
@@ -1,27 +1,27 @@
-package dev.openfeature.contrib.providers.flagd.resolver.process.storage.connector.grpc;
+package dev.openfeature.contrib.providers.flagd.resolver.process.storage.connector.sync;
 
 import dev.openfeature.flagd.grpc.sync.Sync.SyncFlagsResponse;
 import lombok.Getter;
 
 @Getter
-class GrpcResponseModel {
+class SyncResponseModel {
     private final SyncFlagsResponse syncFlagsResponse;
     private final Throwable error;
     private final boolean complete;
 
-    public GrpcResponseModel(final Throwable error) {
+    public SyncResponseModel(final Throwable error) {
```

Comment:
All of our "Sync" related stuff is now mostly prefixed with "Sync" instead of "Grpc" which is pretty ambiguous since both modes use grpc.